Makefile

```diff
@@ -7,6 +7,9 @@ ifeq ($(GOARCH),arm)
 else
 	ARCH=$(GOARCH)
 endif
+# TODO remove GODEBUG=gotypesalias=0
```

Comment:
will we be able to remove it? IIUC the default value changed to 1 in Go 1.23 and it's not going to be reverted to 0?

Reply:
We can remove it when all code generation tools are upgraded to `gengo v2` version.
ref: https://github.com/kubernetes/gengo/pull/281